checks/apps/gromacs/gromacs_benchpep-h.py

```diff
@@ -0,0 +1,162 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
+@rfm.simple_test
+class lumi_gromacs_pep_h(gromacs_check):
+    benchmark_info = parameter([
+        ('benchPEP-h', -1.08735e+06, 0.001), 
+    ], fmt=lambda x: x[0], loggable=True)
+    nb_impl = parameter(['gpu'])
+    maintainers = ['mszpindler']
+    use_multithreading = False
+    exclusive_access = True
+    num_nodes = parameter([1,2], loggable=True)
+    num_gpus_per_node = 8
+    num_tasks_per_node = 8
+    time_limit = '15m'
+    valid_systems = ['lumi:gpu']
+    valid_prog_environs = ['cpeGNU']
+    modules = ['GROMACS/2023.2-cpeGNU-22.12-HeFFTe-GPU']
+
+    allref = {
+        1: {
+            'gfx90a': {
+                'benchPEP-h': (7.3, -0.05, 0.05, 'ns/day'),
+            },
+        },
+        2: {
+            'gfx90a': {
+                'benchPEP-h': (13.2, -0.05, 0.05, 'ns/day'),
+            },
+        },
+    }
+
+    @loggable
+    @property
+    def bench_name(self):
+        '''The benchmark name.
+
+        :type: :class:`str`
+        '''
+
+        return self.__bench
+
+    @property
+    def energy_ref(self):
+        '''The energy reference value for this benchmark.
+
+        :type: :class:`str`
+        '''
+        return self.__nrg_ref
+
+    @property
+    def energy_tol(self):
+        '''The energy tolerance value for this benchmark.
+
+        :type: :class:`str`
+        '''
+        return self.__nrg_tol
+
+    @run_after('init')
+    def prepare_test(self):
+        self.__bench, self.__nrg_ref, self.__nrg_tol = self.benchmark_info
+        self.descr = f'GROMACS {self.__bench} benchmark (NB: {self.nb_impl})'
+        #self.prerun_cmds = [
+        #    f'curl -LJO https://www.mpinat.mpg.de/{self.__bench} && unzip {self.__bench}.zip && ln -s {self.__bench}.tpr benchmark.tpr'
+        #]
+        bench_file_path = os.path.join(self.current_system.resourcesdir, 'gromacs-benchmarks', 'www.mpinat.mpg.de', self.__bench, f'{self.__bench}.tpr') 
+        self.prerun_cmds = [
+            f'ln -s {bench_file_path} benchmark.tpr'
+        ]
+        npme_ranks = 2*self.num_nodes
+        self.executable_opts += [
+            '-nsteps 5000',
+            '-nstlist 400',
+            '-noconfout',
+            '-nb', self.nb_impl,
+            f'-npme {npme_ranks}',
+            '-pme', self.nb_impl, 
+            '-update', self.nb_impl, 
+            '-bonded', self.nb_impl, 
```

Comment:
Some of these options may be useful to vary to trigger the main use-cases of the parallel engine:
- `-update cpu` will switch from GPU-resident mode to force offload mode (useful to test as some input options or settings may prevent `-update gpu`)
- `-bonded cpu` (with `-update gpu`) to test at least one GPU-resident run with CPU contributions.